dependency_support/load_external.bzl

```diff
@@ -295,12 +295,12 @@ def load_external_repositories():
         urls = ["https://github.com/grailbio/bazel-compilation-database/archive/940cedacdb8a1acbce42093bf67f3a5ca8b265f7.tar.gz"],
     )
 
-    # Tagged 2024-06-19, current as of 2024-06-26
+    # Current as of 2024-08-23
     http_archive(
         name = "verible",
-        sha256 = "8c5d13bdc4f4bc756b0f6d31b61cc8aba75dfcfcf22822244ac9c0ccb43db814",
-        strip_prefix = "verible-0.0-3704-g1d393d43",
-        urls = ["https://github.com/chipsalliance/verible/archive/refs/tags/v0.0-3704-g1d393d43.tar.gz"],
+        sha256 = "0f6e2f0cad335c9aca903aaa014f2a55d40c5c4dbfe4666474771f5c08e4a42c",
```

Comment:
Verible upstream with the needed change is 
https://github.com/chipsalliance/verible/releases/tag/v0.0-3754-g17e909b0

Reply:
Switched to use the tag instead of the commit.